adapter/internal/oasparser/model/api_yaml.go

```diff
@@ -104,21 +105,22 @@ type OperationPolicies struct {
 	Fault    PolicyList `json:"fault,omitempty"`
 }
 
-// policyStats used to optimize and reduce loops by storing stats by calculating only once
-type policyStats struct {
-	firstIndex int
-	count      int
-}
-
 // PolicyList holds list of Polices in a flow of operation
 type PolicyList []Policy
 
 // Policy holds APIM policies
 type Policy struct {
-	PolicyName string      `json:"policyName,omitempty"`
-	Action     string      `json:"-"`
-	Order      int         `json:"order,omitempty"`
-	Parameters interface{} `json:"parameters,omitempty"`
+	PolicyName    string      `json:"policyName,omitempty"`
+	PolicyVersion string      `json:"policyVersion,omitempty"`
+	Action        string      `json:"-"`
+	Order         int         `json:"order,omitempty"`
+	Parameters    interface{} `json:"parameters,omitempty"`
+}
+
+// GetFullName returns the fully qualified name of the policy
+// This should be equal to the policy spec/def file name
+func (p *Policy) GetFullName() string {
```

Comment:
Can we rename the method to `GetFullPolicyName` ?

Reply:
This is a function of a policy struct anyway, feels `GetFullName`is enough. @suksw what is ur opinion on this?

Reply:
+1 for `GetFullName` to avoid redundancy, since the struct is already `Policy`